slackbridge/bots.py

```diff
@@ -150,6 +159,21 @@ def nickserv_auth(self):
                 self.nickserv_password,
             ))
 
+    def privmsg(self, user, channel, message):
+        """
+        Handler if a private message is received
+        by an IRC user bot and addressed to self
+        """
+        if channel == self.nickname:
+            if not self.im_id:
+                self.im_id = self.sc.api_call(
```

Comment:
so this method opens a private channel in Slack between the bot and the user. I'm caching the im_id here to avoid making extraneous calls every message (which are also rate-limited).